internal/agent/validate.go

```diff
@@ -0,0 +1,66 @@
+package agent
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	schema "github.com/kairos-io/kairos/pkg/config/schemas"
+)
+
+func JSONSchema(version string) error {
+	url := fmt.Sprintf("https://kairos.io/%s/cloud-config.json", version)
+	schema, err := schema.GenerateSchema(schema.RootSchema{}, url)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(schema)
+
+	return nil
+}
+
+func Validate(file string) error {
+	var yaml string
+
+	if strings.HasPrefix(file, "http") {
+		resp, err := http.Get(file)
+		if err != nil {
+			return err
+		}
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		//Convert the body to type string
+		yaml = string(body)
+	} else {
+		dat, err := os.ReadFile(file)
```

Comment:
should we check if its a dir? Will this even work if we pass a dir with the cloud config on a file in it?

Reply:
Good question, the use case I thought was: "hey let me check if this particular config is valid" just like what we do on the WebUI, but I can definitely extend it to pass dirs too